Patches/Blastech Industries/Patch_Guns.xml

```diff
@@ -0,0 +1,880 @@
+				<Properties>
+					<recoilAmount>1.73</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_Blaster_Blue_Mid</defaultProjectile>
+					<warmupTime>0.6</warmupTime>
+					<burstShotCount>6</burstShotCount>
+					<ticksBetweenBurstShots>9</ticksBetweenBurstShots>
+					<range>55</range>
+					<soundCast>DC-15A_Blaster_Rifle_Shot</soundCast>
+					<muzzleFlashScale>9</muzzleFlashScale>
+					<soundCastTail>GunTail_Light</soundCastTail>
+				</Properties>
+				<FireModes>
+					<aimedBurstShotCount>3</aimedBurstShotCount>
+					<aiUseBurstMode>FALSE</aiUseBurstMode>
+					<aiAimMode>AimedShot</aiAimMode>
+				</FireModes>
+		</li>
+		
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+			<defName>DC-17M_Repeating_Blaster_Rifle</defName>
+				<statBases>
+					<SightsEfficiency>1.0</SightsEfficiency>
+					<ShotSpread>0.14</ShotSpread>
+					<SwayFactor>1.34</SwayFactor>
+					<Bulk>7.20</Bulk>
+					<Mass>3.5</Mass>
+					<RangedWeapon_Cooldown>.5</RangedWeapon_Cooldown>
+				</statBases>
+				<Properties>
+					<recoilAmount>1.48</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_Blaster_Blue_Mid</defaultProjectile>
+					<warmupTime>0.8</warmupTime>
+					<burstShotCount>5</burstShotCount>
+					<ticksBetweenBurstShots>10</ticksBetweenBurstShots>
+					<range>30</range>
+					<soundCast>DC-17_Blaster_Pistol_Shot</soundCast>
+					<muzzleFlashScale>9</muzzleFlashScale>
+					<soundCastTail>GunTail_Heavy</soundCastTail>
+				</Properties>
+				<FireModes>
+					<aimedBurstShotCount>2</aimedBurstShotCount>
+					<aiUseBurstMode>FALSE</aiUseBurstMode>
+					<aiAimMode>AimedShot</aiAimMode>
+				</FireModes>
+		</li>
+		
+
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+			<defName>DC15x_Sinper_Rifle</defName>
+				<statBases>
+					<SightsEfficiency>2.3</SightsEfficiency>
+					<ShotSpread>0.04</ShotSpread>
+					<SwayFactor>1.7</SwayFactor>
+					<Bulk>12.25</Bulk>
+					<Mass>5</Mass>
+					<RangedWeapon_Cooldown>2</RangedWeapon_Cooldown>
+				</statBases>
+				<Properties>
+					<recoilAmount>1.48</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_Blaster_Blue_High</defaultProjectile>
+					<warmupTime>3</warmupTime>
+					<range>75</range>
+					<soundCast>38x-targeting-rifle-sound-effects-battlefront-ii</soundCast>
+					<muzzleFlashScale>10</muzzleFlashScale>
+					<soundCastTail>GunTail_Heavy</soundCastTail>
+				</Properties>
+				<FireModes>
+					<aiAimMode>AimedShot</aiAimMode>
+				</FireModes>
+		</li>
+
+
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+			<defName>Z6_Rotary_Blaster_Cannon</defName>
+				<statBases>
+					<SightsEfficiency>2.3</SightsEfficiency>
```

Comment:
This is awfully high for a heavy repeating weapon.

Reply:
Fixed the typo